docs: adding PD manual (mkdocs) (#18833)
OpenTrons/opentrons-api: docs/pd-manual/docs/starting-deck.md

Click **Edit protocol** to add and edit hardware, labware,  and liquids on the protocol starting deck. 

> The protocol starting deck view shows how your Flex or OT-2 deck will look at the beginning of a protocol. This is the first step in your protocol timeline, shown on the left. It's also the same view of the deck as in the protocol overview, but now allows you to edit. 

Reviewer: Something wrong about this syntax… this is simpler:
```suggestion
The protocol starting deck view shows how your Flex or OT-2 deck will look at the beginning of a protocol. This is the first step in your protocol timeline, shown on the left. It's also the same view of the deck as in the protocol overview, but editable. 
```

> Deck Hardware
> --------------

Reviewer: While you _can_ use underline headers in markdown, they're considered pretty nonstandard now. Let's use hashes. 
```suggestion
## Deck hardware
```

<figure class="screenshot" markdown>
  ![Configure hardware](images/configure_hardware.png)
  <figcaption>Add, move, or delete deck hardware, like modules and fixtures.</figcaption>
  </figure>

Liquids
---------
Next, click **Liquids** in the upper left. First, define a liquid to use in your protocol with a name, description, and color. You can also define the liquid as an Opentrons-verified liquid class to apply optimized pipetting settings during transfer and mix steps. You'll be able to add liquids to labware already on the robot deck.

Labware
--------
Click to return to the protocol starting deck, the first step in your protocol timeline. Here, you can edit labware on the robot deck. Hover over each deck slot to view slot details. Then, click individual slots to add, remove, or change labware. 

<figure class="screenshot" markdown>
  ![Add labware](images/add_labware3.png)
  <figcaption>Add a well plate to the deck.</figcaption>
  </figure>

> For the example slot D2, you can add labware, like a NEST well plate, by clicking the deck slot and **Add labware**. In the menu, search for or select a labware type, like reservoirs or well plates, and view available options from the [Labware Library](https://labware.opentrons.com "Labware Library"). At the bottom of the labware list, you can upload a JSON file to use custom labware in your protocol. 

Reviewer: This paragraph offers multiple conflicting suggestions: "like a NEST well plate", "like reservoirs", "you can…use custom labware". Either present a single use case (or, possibly, no use case).

Author: Simplified this.

Click **Edit liquid** in the labware menu on the right to add liquids you've already defined in your protocol. In the labware graphic, click and drag to select wells across rows and columns. From the dropdown menu, select your liquid and enter the starting volume for each well in microliters (µL). Click **Save** for each liquid added to your labware before clicking **Done**.  

<figure class="screenshot" markdown>
  ![Add liquid](images/add_liquid.png)
  <figcaption>Add liquid to your chosen wells. Wells shown in blue are selected to add a second liquid (plasmid #2). </figcaption>
  </figure>

> Click a module on the deck and choose **Edit labware** to add labware. Protocol Designer only shows recommended labware in the list of available options. You can choose to view all labware, including labware that may be incompatible with the module. Add compatible adapters for modules on the deck from the "adapter" labware category. Use the toggle switch at the upper right of the protocol starting deck if you need to add any off-deck labware. 

Reviewer: Needs a transition, header, additional screenshot, or some combination here. Otherwise it feels like this goes along with the labware detail view (where you add liquid), but that's the wrong context for these actions.

Author: Added a combination here.
